project 1/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
```

Comment:
The folder for this project needs to be renamed -- to not include spaces, e.g. `project1/` not `project 1/`